crates/core/tedge_actors/src/tasks.rs

```diff
@@ -0,0 +1,49 @@
+use crate::Actor;
+use crate::RuntimeError;
+use async_trait::async_trait;
+use std::fmt::Formatter;
+
+/// A task to be spawn in the background
+#[async_trait]
+pub trait Task: 'static + Send + Sync {
+    fn name(&self) -> &str;
+    async fn run(self: Box<Self>) -> Result<(), RuntimeError>;
+}
```

Comment:
Is this trait useful? It's used only in one place.

One can consider merging this trait with the `Actor` trait.

Reply:
This can be remove. The motivation was to abstract the async runtime (tokio or futures or ...) behind the actor runtime. For now we will simply use tokio directly and this trait is a bit useless.

Reply:
This will be removed as a follow-up task.